### PR TITLE
#24 add data cite doi support

### PR DESCRIPTION
**Pull request by sekmiller**
Updated Documentation for Installers Guide included

### CODE REVIEW

**Comment by sekmiller** [comment]
Related Issues:
#24 

RFI Checklist

- [x ]  Functionality completed as described in FRD
- [x]  Dependencies, risks, assumptions in FRD addressed
- [ ]  Unit tests completed
- [x]  Deployment requirements identified (e.g., SQL scripts, indexing)
- [x]  Documentation completed
- [x]  All code checkins completed
<hr>
- [ ]  Code review completed or waived
- [ ]  Testing requirements completed
- [ ]  Usability testing completed or waived
- [ ]  Support testing completed or waived
- [ ]  Merged with develop branch and resolved conflicts

**Comment by eaquigley** [comment]
Waiving usability testing as it is not necessary for this functionality.

**Comment by pdurbin** [comment]
I see docs have already been marked as complete but @posixeleni just pointed out that DataCite customers can subscribe to updates at http://status.datacite.org and I guess we **could** mention this in our docs. Just a suggestion, really. Perhaps someone could add it to the list at #2944.

**Comment by pdurbin** [comment]
@sekmiller as we discussed, please update https://github.com/IQSS/dataverse/blob/24-add-DataCite-DOI-Support/doc/sphinx-guides/source/installation/prep.rst#architecture-and-components

If you'd like me to help you create a dedicated page for "Persistent indentifer service" (whoops, please fix that typo while you're in there), please let me know.

**Comment by sekmiller** [comment]
Handing back to Gustavo for code review.  Will make a small edit to documentation as noted by Phil above.

**Comment by kcondon** [comment]
Found several issues needing attention:
#2981, #2983, #2988, #2989, #2990

**Comment by sekmiller** [comment]
@scolapasta The conflict is most likely on the Dataset.xhtml.  Per our discussion about the warning we show to users if the publish is going to also register the dataset, @eaquigley asked me to remove it completely (for DataCite and EZID instances) - it's a sausage-making message that the users don't care about.  They should be aware that publishing a dataset will include having a public doi.

**Comment by sekmiller** [comment]
Checkins completed.  Ready for testing

**Comment by kcondon** [comment]
OK to merge

File: doc/sphinx-guides/source/installation/prep.rst [hunk 1]
```diff
@@ -56,7 +56,7 @@ When planning your installation you should be aware of the following components
 - PostgreSQL: a relational database.
 - Solr: a search engine. A Dataverse-specific schema is provided.
 - SMTP server: for sending mail for password resets and other notifications.
-- Persistent indentifer service: DOI support is provided. An EZID subscription is required for production use. 
+- Persistent indentifier service: DOI support is provided. An EZID subscription or DataCite account is required for production use.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/installation/prep.rst]: Gah! It's still spelled wrong! From "indentifer" to "indentifier". D'oh!

**Pull request merged**